kubernetes-elasticsearch-cluster/deploy-es.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+if (( $# != 1 ))
+then
+  echo "Usage: kubernetes-elasticsearch-cluster/deploy-es.sh <dataset>"
+  echo "  where <dataset> is the name of a directory in dataset_config/"
+  echo "Run this script from project root"
+  exit 1
+fi
+
+dataset=$1
+project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
+gcloud config set project $project_id
+zone=$(gcloud container clusters list | grep elasticsearch-cluster | awk '{print $2}')
+gcloud container clusters get-credentials elasticsearch-cluster --zone ${zone}
+
+# Note that "node" here refers to ES node, not GKE node.
+node_pool_machine_type=$(jq --raw-output '.node_pool_machine_type' dataset_config/${dataset}/deploy.json)
+node_pool_num_nodes=$(jq --raw-output '.node_pool_num_nodes' dataset_config/${dataset}/deploy.json) || "3"
```

Comment:
Delete: || "3" ?

Seems unnecessary because of 28

Reply:
Yup oops, done.